### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by LazarAndrei1106**

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -46,9 +46,17 @@ La final, o să fii redirecționat înapoi pe [**UCP**](https://ucp.liberty.mp/)
 
 ## Cum îmi asociez contul din joc cu o adresă de E-mail?
 
+Pentru a îți asocia contul din joc cu o adresa de E-mail trebuie să te autentifici pe [**UCP**](https://ucp.liberty.mp/), să apeși pe numele tău din dreapta-sus, apoi să selectezi **Settings**.
```
Comment (leonardssh) [inline faq.md:49]:
```suggestion
Pentru a îți asocia contul din joc cu o adresă de E-mail trebuie să te autentifici pe [**UCP**](https://ucp.liberty.mp/), să apeși pe numele tău din dreapta-sus, apoi să selectezi **Settings**. 
```

File: faq.md [hunk 2]
```diff
@@ -46,9 +46,17 @@ La final, o să fii redirecționat înapoi pe [**UCP**](https://ucp.liberty.mp/)
 
 ## Cum îmi asociez contul din joc cu o adresă de E-mail?
 
+Pentru a îți asocia contul din joc cu o adresa de E-mail trebuie să te autentifici pe [**UCP**](https://ucp.liberty.mp/), să apeși pe numele tău din dreapta-sus, apoi să selectezi **Settings**. 
+
+Ulterior, apasă pe opționea **Set** din dreptul textlui **Email address**. Introduci adresa de email dorita apoi selectezi **Set email**. O sa primești un mail de confirmare pe adresa de email introdusa.
```
Comment (leonardssh) [inline faq.md:51]:
```suggestion
Ulterior, apasă pe opțiunea **Set** din dreptul textului **Email address**. Introduci adresa de email dorită apoi selectezi **Set email**. O sa primești un mail de confirmare pe adresa de email introdusă.
```

File: faq.md [hunk 3]
```diff
@@ -46,9 +46,17 @@ La final, o să fii redirecționat înapoi pe [**UCP**](https://ucp.liberty.mp/)
...
+
+Dupa accesarea linkului de confirmare, o să fii redirecționat înapoi pe [**UCP**](https://ucp.liberty.mp/) unde îți va fi afișat în stânga-jos un mesaj prin care te informează că acțiunea de asociere a avut succes.
```
Comment (leonardssh) [inline faq.md:53]:
```suggestion
După accesarea linkului de confirmare, o să fii redirecționat înapoi pe [**UCP**](https://ucp.liberty.mp/) unde îți va fi afișat în stânga-jos un mesaj prin care te informează că acțiunea de asociere a avut succes.
```

File: faq.md [hunk 4]
```diff
@@ -46,9 +46,17 @@ La final, o să fii redirecționat înapoi pe [**UCP**](https://ucp.liberty.mp/)
...
+
+<Image src="https://im5.ezgif.com/tmp/ezgif-5-9958a75010.gif" alt="Link Email" />
```
Comment (leonardssh) [inline faq.md:55]: The gif doesn't work.

File: faq.md [hunk 5]
```diff
@@ -46,9 +46,17 @@ La final, o să fii redirecționat înapoi pe [**UCP**](https://ucp.liberty.mp/)
...
+
 ::: warning
-Momentan, funcția de asociere a unui cont din joc cu o adresă de **E-mail** nu este disponibilă. :slightly_frowning_face: 
-:::
+Daca mail-ul de confirmare nu apare in **Inbox** sugeram sa il cauti in **Spam**.
```
Comment (leonardssh) [inline faq.md:58]:
```suggestion
Dacă mail-ul de confirmare nu apare in **Inbox**, îți sugerăm să-l cauți in **Spam**.
```

**Review by leonardssh — CHANGES_REQUESTED** [review]

File: faq.md (hunk 4 above)
Comment (LazarAndrei1106) [reply faq.md:55]: Nu cred :( la mine mergea.

**Review by leonardssh — APPROVED** [review]

**Comment by leonardssh** [comment]
This PR needs a GIF.

**Pull request closed without merging**